stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/dto/UserDtos.kt

```diff
@@ -84,3 +84,10 @@ internal data class PartialUpdateUserDto(
     val set: Map<String, Any>,
     val unset: List<String>,
 )
+
+@JsonClass(generateAdapter = true)
+internal data class DownstreamUserBlockDto(
+    val blocked_by_user_id: String,
+    val blocked_user_id: String,
+    val created_at: String,
```

Comment:
On Dtos we normally use `ExactDate` for dates

Reply:
Used `Date` to be consistent in the file.